src/css/Blog.css

```
@@ -7,8 +7,241 @@
+}
+
+.reply-by-email {
+	color: var(--color-text-muted);
+	font-size: 0.9rem;
+	font-weight: 300;
+}
+
+.reply-by-email a {
+	color: var(--color-accent);
+	text-decoration: none;
+	border-bottom: 1px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)